test/api/v3/unit/libs/email.test.js

```diff
@@ -137,6 +155,29 @@ describe('emails', () => {
     });
   });
 
+  describe('getGroupUrl', () => {
+    it('returns correct url if group is the tavern', () => {
+      let attachEmail = requireAgain(pathToEmailLib);
+      let getGroupUrl = attachEmail.getGroupUrl;
+      let group = getGroup('tavern');
+      expect(getGroupUrl(group)).to.eql('/#/options/groups/tavern');
+    });
+
+    it('returns correct url if group is a guild', () => {
+      let attachEmail = requireAgain(pathToEmailLib);
+      let getGroupUrl = attachEmail.getGroupUrl;
+      let group = getGroup('guild');
+      expect(getGroupUrl(group)).to.eql(`/#/options/groups/guilds/${group._id}`);
+    });
+
+    it('returns correct url if group is a party', () => {
```

Comment:
Technically speaking, this `returns the party url for any group that is not a guild`